src/content/getusermedia/record/js/main.js

```diff
@@ -84,21 +81,26 @@ function startRecording() {
   recordedBlobs = [];
   let options = {mimeType: 'video/webm;codecs=vp9'};
   if (!MediaRecorder.isTypeSupported(options.mimeType)) {
-    console.log(`${options.mimeType} is not Supported`);
+    console.error(`${options.mimeType} is not Supported`);
+    errorMsgElement.innerHTML = `${options.mimeType} is not Supported`;
```

Comment:
Still think that those 2 lines could be wrapped into a helper, but I'll let that pass.